2021/day14/day14.java

```diff
@@ -0,0 +1,88 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.io.File;
+
+public class day14 {
+    public static class Rule {
+        String pair;
+        String outcome;
+    }
+    // Read polymer template e.g. NCNBCHB
```

Comment:
This is not descriptive - the name of the method already mentions all of this. I would say this method does not require a description